guidelines/sc/22/focus-appearance-minimum.html

```diff
@@ -24,8 +24,7 @@ <h4>Focus Appearance (Minimum)</h4>
             </ul>
         </li>
     </ul>
+                
+    <p>Where a user interface component has active sub-components, if a sub-component receives a focus indicator, these requirements can be applied to the sub-component instead.</p>
```

Comment:
Is the note supposed to appear on the index page as well? If so it needs to be here.